test/full_url_for.spec.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+describe('full_url_for', () => {
+  const ctx = {
+    config: {}
+  };
+
+  const fullUrlFor = require('../lib/full_url_for').bind(ctx);
+
+  it('internal url - root directory', () => {
+    ctx.config.root = '/';
```

Comment:
`config.root` is never used, so there's no need to test it.

Reply:
Removed.